nrfcloud/calculateCostsPerAccount.spec.ts

```diff
@@ -0,0 +1,121 @@
+import { STACK_NAME } from '../cdk/stacks/stackConfig.js'
+import { SSMClient } from '@aws-sdk/client-ssm'
+import { calculateCostsPerAccount } from './calculateCostsPerAccount.js'
+const testDateAugust = 1691145383000
```

Comment:
```suggestion
const testDateAugust = new Date('2023-08-04T10:36:23.000Z')
```
is much more readable.